README.md

```diff
@@ -23,13 +23,22 @@ courts.get_selectable() # returns a list of all courts that are whitelisted to
 
 courts.get_listable_groups() # returns a grouped list of courts that are whitelisted to
```

Comment:
I think this could do with a slight wording tweak to clarify that these could be courts or tribunals. "courts/tribunals" should do it.

Reply:
aha yep i was wondering about that - it's now unclear because "courts" is used interchangably. I'll see if i can make that clearer!